sopel/cli/plugins.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
+"""Sopel Plugins Command Line Interface (CLI): ``sopel-plugins``"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import argparse
+
+from sopel import plugins, tools
+
+from . import utils
+
+
+def build_parser():
+    """Configure an argument parser for ``sopel-plugins``"""
+    parser = argparse.ArgumentParser(
+        description='Sopel plugins tool')
+
+    # Subparser: sopel-plugins <sub-parser> <sub-options>
+    subparsers = parser.add_subparsers(
+        help='Actions to perform',
+        dest='action')
+
+    # sopel-plugins list
+    list_parser = subparsers.add_parser(
+        'list',
+        help="List available Sopel plugins",
+        description="""
+            List available Sopel plugins from all possible sources: built-in,
+            from ``sopel_modules.*``, from ``sopel.plugins`` entry points,
+            or Sopel's plugin directories. Enabled plugins are displayed in
+            green, disabled in red.
```

Comment:
I forgot to add this to my earlier review. >.<

```suggestion
            green; disabled, in red.
```